cookbook/console/command_in_controller.rst

```diff
@@ -0,0 +1,94 @@
+.. index::
+   single: Console; Use commands in your Controller
+
+How to Call a Command from a Controller
+=======================================
+
+The :doc:`Console component documentation </components/console/introduction>` covers
+how to create a console command. This cookbook article covers how to use a console command
+directly in your application. 
+
+You may have the need to execute some function that is only available in a console command. 
+Usually, you should refactor the command and move some logic into a service that can be 
+reused in the controller. However, when the command is part of a third-party library, you 
+wouldn't want to modify or duplicate their code, but want to directly execute the command 
+instead. 
+
+An example of this is sending the emails that Swift Mailer spooled earlier
+:doc:`using the swiftmailer:spool:send command </cookbook/email/spool>`. Symfony
+allows you to directly execute a registered ``Command`` inside your Controller::
+
+    // src/AcmeBundle/Controller/SpoolController.php
+    namespace AcmeBundle\Controller;
```

Comment:
I think we should adapt the best practices here and use the `AppBundle` namespace instead.